option optimize_for = LITE_RUNTIME;
import "U64Type.proto";
message HeroSellRequest
{
 repeated U64Type guid = 1;
}
